caluma/form/tests/test_document.py

```diff
@@ -875,3 +875,33 @@ def filter_queryset_for_document(self, node, queryset, info):
         assert result.data["documentValidity"] is None
     else:
         assert len(result.data["documentValidity"]["edges"]) == 1
+
+
+@pytest.mark.parametrize("remove_case", [True, False])
+def test_remove_document(db, remove_case, document, answer, case, schema_executor):
```

Comment:
This parametrisation is so invasive, I figure it'd be cleaner to have two separate tests for this

Reply:
Done.